### PR TITLE
BitwiseAnd, BitwiseOr, BitwiseXor support

### PR DESCRIPTION
**Pull request by philass**
### Description

This PR adds BitwiseAnd, BitwiseOr, and BitwiseXor

### Motivation and Context

There is no way to sanely do bitwise operations with the current ONNX opset. Relates to https://github.com/onnx/onnx/issues/4343

### Additional comments

Each Bitwise op has 4 tests.

Also I was in two minds if I should include `BitwiseNot` in this PR, or in another PR. If you guys feel it should be included in this one please let me know. The PR for `BitwiseNot` is here https://github.com/onnx/onnx/pull/4497

### CODE REVIEW

File: onnx/defs/logical/defs.cc [hunk 1]
```diff
@@ -219,4 +219,99 @@ ONNX_OPERATOR_SET_SCHEMA(
         }
         )ONNX"));
 
+std::function<void(OpSchema&)> BinaryBitwiseDocGenerator(const char* name) {
+  return [=](OpSchema& schema) {
+    std::string doc;
+    POPULATE_OP_DOC_STR(doc = R"DOC(
+Returns the tensor resulted from performing the `{name}` bitwise operation
```
Comment (gramalingam) [inline onnx/defs/logical/defs.cc:226]: Nit:
(i) change `resulted` to `resulting`. 
(ii) exchange the order of `{name}` and `bitwise`.

**Comment by gramalingam** [comment]
LGTM, thanks!

File: onnx/defs/logical/defs.cc (hunk 1 above)
Comment (philass) [reply onnx/defs/logical/defs.cc]: Good catch! That is definitely better wording

I've made the changes to the PR

**Review by p-wysocki — APPROVED** [review]
Would testing 0d tensor inputs (scalars, essentially) bring any benefits? Overall LGTM, that is very clean code.

**Comment by philass** [comment]
@p-wysocki thanks for the review!

I checked other unary and binary ops. And it seems that most times 0d tensors aren't checked.

If you feel it would add more coverage, then I am happy to add tests.

**Review by gramalingam — APPROVED** [review]

**Comment by gramalingam** [comment]
I think some minor conflict-resolution is required since the other PR (NOT op) touches the same lines in the opset file.

**Comment by philass** [comment]
> I think some minor conflict-resolution is required since the other PR (NOT op) touches the same lines in the opset file.

Thanks for merging in `BitwiseNot`. I have fixed the merge conflicts. And these ops should be ready to merge.

**Pull request merged**